source/content/terminus/install.md

```diff
@@ -42,11 +42,11 @@ The following commands will:
 - add a symlink to your local `bin` directory for the Terminus executable.
 
   ```bash{promptUser: user}
-  mkdir ~/terminus && cd ~/terminus
-  TERMINUS_RELEASE=$(curl --silent "https://api.github.com/repos/pantheon-systems/terminus/releases/latest" | perl -nle'print $& while m#"tag_name": "\K[^"]*#g')
-  curl -L https://github.com/pantheon-systems/terminus/releases/download/$TERMINUS_RELEASE/terminus.phar --output terminus
-  chmod +x terminus
-  sudo ln -s ~/terminus/terminus /usr/local/bin/terminus
+mkdir -p ~/terminus && cd ~/terminus
+curl -L https://github.com/pantheon-systems/terminus/releases/download/3.0.0/terminus.phar --output terminus
```

Comment:
Shouldn't this be `3.0.5` instead of `3.0.0`?  Otherwise, `self:update` doesn't work, right?

Reply:
Thank you so much @paulsheldrake, @uberhacker, and @kporras07! We really appreciate all your work on this 

Looking over what Kevin previously added, @uberhacker I think you are correct. @kporras07 can you please confirm?

Reply:
Yeah, let's make a 3.0.6 release f Terminus, and then use that here.

Reply:
Great! Thanks a million 💯

Reply:
Hey @greg-1-anderson could you please review - this has been updated to `3.0.6`